integration/test_ft_alias_cluster.py

```
@@ -0,0 +1,2087 @@
+            node0.execute_command(
+                "FT._DEBUG PAUSEPOINT RESET fanout_remote_pausepoint")
+            node1.execute_command(
+                "FT._DEBUG CONTROLLED_VARIABLE SET PauseHandleClusterMessage no")
+        else:
+            # Safety fallback.
+            node0.execute_command(
+                "FT._DEBUG PAUSEPOINT RESET fanout_remote_pausepoint")
+            node1.execute_command(
+                "FT._DEBUG CONTROLLED_VARIABLE SET PauseHandleClusterMessage no")
+            exception = ValueError(f"Invalid order: {order}")
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)